user/plan-sharing.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Plan Sharing Guide
+layout: en
+
+---
+
+Assembla’s billing model allows customers to pay once for an entire portfolio of projects and allocate the allowance across the projects. Travis CI modified the billing model to map Assembla’s spaces to Travis CI Organizations and fulfill customer needs.  
+
+Travis CI introduces a flexible new feature that allows users to share paid plans across Assembla’s spaces. This innovation introduces two new roles to Travis CI plans: the Donor and the Receiver, providing users with more control over their resources. 
+
+## Donor and Receiver 
+Travis CI users with paid plans can enable the Plan Sharing feature and decide to share with the available Spaces.
+
+A user who enables the Sharing Plan option is referenced as the *** Donor *** account. 
+
+A ***Donor*** account consists of a Travis CI user or Organization willingly sharing a Travis CI Plan allowance with other Travis CI users or Organizations. 
+
+A user who benefits from the Sharing Plan feature is referenced as the ***Receiver*** account. 
+
+A ***Receiver*** account is a Travis CI user or Organization that benefits from accessing a Travis CI Shared Plan by a Donor account. 
+ 
+## How does a Shared Plan work?
+Only Travis CI paid plans are available for sharing with another Travis CI Organization linked with an Assembla Space.
+
+All recipients may trigger builds within the allowances of the original plan. It means that the system tracks all metrics under the original plan, but there are now multiple sources for these metrics.
+
+### Concurrency-based Plans
+Concurrency-based plans are available to all Travis CI accounts that use the Shared Plan, including Donor and Receiver accounts. When sharing a concurrency-based plan, the number of jobs is shared between all accounts. For example, if user account ***A*** decides to share a plan with 20 concurrent jobs with user accounts ***B*** and ***C***. This means that all three accounts have a total of 20 concurrent jobs. 
+
+Here is an example:
+User account ***A*** runs a build with 10 concurrent jobs. This consumes 10 of the 20 available slots. Then, Account ***B*** runs a build with 7 concurrent jobs, which consumes 17 out of the 20 available slots. Finally, if account ***C*** runs a build with 5 or more concurrent jobs, only 3 jobs are available to start, while the remaining jobs must wait in the queue for free slots.  
```

Comment:
ditto re 'user account' - Donor/Receiver or Travis account, depending on the context